src/stravalib/tests/integration/test_client.py

```diff
@@ -157,18 +165,38 @@ def test_get_club_activities(mock_strava_api, client):
         response_update={"distance": 1000},
         n_results=2,
     )
+
     activities = list(client.get_club_activities(42))
     assert len(activities) == 2
     assert activities[0].distance == meters(1000)
 
 
-def test_get_activity_zones(mock_strava_api, client):
-    # Unfortunately, there is no example response for this endpoint in swagger.json
-    with open(
-        os.path.join(RESOURCES_DIR, "example_zone_response.json"), "r"
-    ) as zone_response_fp:
-        zone_response = json.load(zone_response_fp)
+def test_get_activity_zones(mock_strava_api, client, zone_response):
+    """Returns an activities associated zone (related to heart rate and power)
+
+    Notes
+    -----
+    There is no example response for this endpoint in swagger.json so
+    we created a sample json file with the return that we are currently
+    seeing. Thus this method could break at any time if Strava changes
+    the response output.
+
+    """
+    #
     mock_strava_api.get("/activities/{id}/zones", json=zone_response)
+    """
+    # TODO: failing here E  pydantic_core._pydantic_core.ValidationError: 10 validation errors for BaseActivityZone
```

Comment:
This is the current failure point in the tests that i could use some guidance on.